packages/store/index.d.ts

```diff
@@ -1,43 +1,45 @@
-declare module drizzle {
-  export class Drizzle {
-    constructor(options: any, store: any);
-
-    addContract(contractConfig: any, ...args: any[]): void;
-
-    deleteContract(contractName: any): void;
-
-    findContractByAddress(address: any): any;
+import {
+  Drizzle,
+  IDrizzleOptions,
+  generateStore,
+  IStoreConfig,
+  generateContractsInitialState,
+} from './types';
+
+export {
+  Drizzle,
+  IDrizzleOptions,
+  generateStore,
+  IStoreConfig,
+  generateContractsInitialState,
+};
+
+export enum EventActions {
+  EVENT_FIRED = 'EVENT_FIRED',
+  EVENT_CHANGED = 'EVENT_CHANGED',
+  EVENT_ERROR = 'EVENT_ERROR',
+}
 
-    generateStore(options: any): any;
+export namespace drizzleReducers {
+  export interface IAction {
+    [key: string]: any;
+    type: string;
   }
 
-  export const EventActions: {
-    EVENT_CHANGED: string;
-    EVENT_ERROR: string;
-    EVENT_FIRED: string;
-  };
-
-  export const drizzleSagas: any[];
-
-  export function generateContractsInitialState(options: any): any;
-
-  export function generateStore(_ref: any): any;
-
-  export namespace drizzleReducers {
-    function accountBalances(...args: any[]): any;
-
-    function accounts(...args: any[]): any;
-
-    function contracts(...args: any[]): any;
-
-    function currentBlock(...args: any[]): any;
-
-    function drizzleStatus(...args: any[]): any;
-
-    function transactionStack(...args: any[]): any;
-
-    function transactions(...args: any[]): any;
+  export function accounts(state: any | undefined | null, action: IAction): any;
+  export function accountBalances(state: any | undefined | null, action: IAction): any;
+  export function contracts(state: any | undefined | null, action: IAction): any;
+  export function currentBlock(state: any | undefined | null, action: IAction): any;
+  export function drizzleStatus(state: any | undefined | null, action: IAction): any;
+  export function transactions(state: any | undefined | null, action: IAction): any;
+  export function transactionStack(state: any | undefined | null, action: IAction): any;
+  export function web3(state: any | undefined | null, action: IAction): any;
+}
 
-    function web3(...args: any[]): any;
-  }
+export namespace drizzleSagas {
```

Comment:
Isn't `drizzleSagas` still an array though https://github.com/trufflesuite/drizzle/blob/develop/packages/store/src/index.js#L36.

Not sure why it was changed to a namespace.